howto/serialize/README.md

```diff
@@ -0,0 +1,27 @@
+# Serialization in Dapr's SDKs

```

Comment:
This today should say java sdk.  Custom serialization is not available in the others.

Reply:
This doc is generic. I will mention that Java is the only one implementing this.